src/test/python/deep_qa_test/models/reading_comprehension/bidirectional_attention_test.py

```diff
@@ -0,0 +1,31 @@
+# pylint: disable=no-self-use,invalid-name
+from unittest import TestCase
+import logging
+import os
+import shutil
+
+from deep_qa.models.reading_comprehension.bidirectional_attention import BidirectionalAttentionFlow
+from deep_qa.common.checks import log_keras_version_info
+from ...common.constants import TEST_DIR
+from ...common.models import get_model, write_span_prediction_files
+from ...common.test_markers import requires_tensorflow
+
+
+class TestBidirectionalAttentionFlow(TestCase):
+    def setUp(self):
+        logging.basicConfig(format='%(asctime)s - %(levelname)s - %(name)s - %(message)s',
+                            level=logging.INFO)
+        log_keras_version_info()
```

Comment:
Actually, I think this should be in all of the tests.  It would make debugging semaphore failures easier.  Once we have a `DeepQaTest` base class, I would put these lines in the `setUp` method there.